charactersheet/charactersheet/templates/character/profile.tmpl.html

```diff
@@ -23,7 +23,9 @@
     	  <div class="input-group">
 	      	<span class="input-group-addon" id="basic-addon2">Alignment</span>
       		<input type="text" class="form-control" aria-describedby="basic-addon2"
-		      	data-bind='textInput: alignment'>
+		      	data-bind="textInput: alignment,
```

Comment:
Do we want to put this here?  So far, we have only put them in the wizard. Up for discussion.